style/ruby/.rubocop.yml

```diff
@@ -90,6 +90,11 @@ Metrics/AbcSize:
                  branches, and conditions.
   Enabled: false
 
+Metrics/BlockLength:
+  CountComments: false  # count full line comments?
+  Max: 25
+  ExcludedMethods: ["describe"]
```

Comment:
Do we need to include nested ones like `context` and `it` as well?

Separately, thoughts excluding the entire `spec` dir? I sometimes have helper methods and such that can get pretty long that may not be worth extracting and having this rule triggered could be annoying